src/Host.cpp

```diff
@@ -63,8 +63,10 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 , mDisplayFont(QFont("Bitstream Vera Sans Mono", 10, QFont::Normal))
 , mEnableGMCP(true)
 , mEnableMSSP(true)
+, mEnableMSP(true)
```

Comment:
I think so.

Reply:
How about a dialog box, such as is done with mapper downloads, that says, "This game offers the Mud Sound Protocol (MSP).  Would you like to enable it?  You could always toggle this on/off from Settings on the Main Toolbar."

How it could work:

When a game sends `WILL MSP` the first time, Mudlet will check options for `mEnableMSP` and find a `-1`.  This combination could guide Mudlet to present the dialog box, then capture a `0` for No or a `1` for Yes to store in `mEnableMSP`.  If a `0` has been set, the next time we receive a `WILL MSP` we could ignore it.  If the game sends us a `DO MSP`, assuming the player toggled something on inside of their game, if we find that `mEnableMSP` is `-1` or `0` we could present the dialog box again.

Reply:
Why do we need to ask? If someone doesn't want it, they can turn it off in settings. I imagine the game itself will also have an option for sounds. 

I feel like everything would be covered :)

Reply:
> How about a dialog box [...?]

@vadi2 You didn't catch this - but @mpconley we do not like to have such pop-ups as: A) they interrupt the (how shall I put it) *user interface* and B) IIRC they can (if they are **modal** dialogues) also interrupt the program execution (from the Qt documentation for the `QDialog` class): 

> #### [Modal Dialogs](https://doc.qt.io/qt-5/qdialog.html#modal-dialogs)
> A **modal** dialog is a dialog that blocks input to other visible windows in the same application. Dialogs that are used to request a file name from the user or that are used to set application preferences are usually modal. Dialogs can be *application modal* (the default) or *window modal*.
>
> When an application modal dialog is opened, the user must finish interacting with the dialog and close it before they can access any other window in the application. Window modal dialogs only block access to the window associated with the dialog, allowing the user to continue to use other windows in an application.

Reply:
Understanding we're all on the same page now that MSP will be enabled by default.  Resolving.